tools/topology/topology2/platform/intel/bt-generic.conf

```diff
@@ -205,12 +205,12 @@ Object.Base {
 
 Object.PCM {
 	pcm."BT" {
-		name		"Port2"
+		name		$BT_PCM_NAME
 		id 		$BT_PCM_ID
 		direction	"duplex"
 
 		Object.Base.fe_dai.1 {
-			name	"Port2"
+			name	$BT_PCM_NAME
```

Comment:
you dont even need this. IT will be inferited from the parent pcm object

Reply:
Ack, will drop.